templates/taxi/driver_detail.html

```diff
@@ -22,5 +22,11 @@ <h4>Cars</h4>
     {% empty %}
       <p>No cars!</p>
     {% endfor %}
+    <hr>
+    <div class="update">
+      <a href="{% url 'taxi:driver-update' driver.id %}" class="btn btn-info">Update</a>
```

Comment:
Ensure that the URL tag syntax is correct. It should be `{% url 'taxi:driver-update' driver.id %}`. Verify that the parameter is correctly passed.